### ISSUE TITLE
Remote Host Closed Connection (0x80070057)

### ISSUE THREAD

**Issue by sam-damico** [Navegate/Navegate#8]
This is a widespread error and could be coming from many places for many reasons. 
This error is specific to the default_v(1-3) pages.

**Comment by sam-damico** [comment]
One place it is happening is in default_v(1-3) when a user spam-clicks the convert to pdf button. I can stop the error from firing by checking if the connection is still alive when it calls Response.flush(). But we should still disable the button to prevent it from taking longer than it has to for a user to get the PDF.

**Comment by shroeder** [comment]
Can we open up a bug for each place this is occuring?

**Comment by sam-damico** [comment]
Yeah, there are a handful of different locations and reasons, i think we should separate them by the error code

**Comment by sam-damico** [comment]
The change I have made now is to still check if the connection is still open before the flush(), but now it will also check if there is already an active connection to the client. If there is it stops a new connection from being made and stopping the old one. This was a simpler solution to the click-happy user problem then disabling and re-enabling the button when the download finished. I have made this change to all three default.ascx.vb pages.

I am assuming this is the only place where this specific error code it coming from so I will close this issue.
